pkgs/data/themes/nixos-bgrt-plymouth/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation {
+  name = "nixos-bgrt-plymouth";
+  version = "unstable-2023-03-10";
+
+  src = fetchFromGitHub {
+    repo = "plymouth-theme-nixos-bgrt";
```

Comment:
Normally we were owner first because it is first in the URL

Reply:
Yeah idk why I wrote it like that this time (because yeah `owner` should really just be first). I'll fix it in the next update. Thanks!